ports/linux/rs485.c

```diff
@@ -607,6 +607,13 @@ void RS485_Initialize(void)
             exit(EXIT_FAILURE);
         }
         /* if all goes well, set new divisor */
+        ioctl(RS485_Handle, TIOCSSERIAL, &newserial);
+    } else if(!RS485_SpecBaud && RS485_Baud == B38400) {
```

Comment:
!RS485_SpecBaud shouldnt be needed here since we are in the else clause of if (RS485_SpecBaud)

Reply:
Very true, good eyes. thanks